google-account-plugin/src/com/google/gct/login/GoogleLogin.java

```diff
@@ -567,6 +567,16 @@ public String getInfo() {
   private class AndroidUiFacade implements UiFacade {
     private GoogleLoginActionButton myButton;
     private volatile CancellableServerReceiver receiver = null;
+    private GoogleLoginMessageExtender messageExtender;
+
+    public AndroidUiFacade() {
+      GoogleLoginMessageExtender[] messageExtenders = Extensions.getExtensions(GoogleLoginMessageExtender.EP_NAME);
+      if (messageExtenders.length == 0) {
+        messageExtender = new DummyGoogleLoginMessageExtender();
+      } else {
+        messageExtender = messageExtenders[0];
```

Comment:
At that time it seemed reasonable to expect only one extension to provide messages. We can however allow any number, though their order of execution will be arbitrary initially (the order IJ/AS returns  the corresponding extensions) as I see no point in implementing some kind of priority logic currently.